Sources/SwiftIfConfig/BuildConfiguration.swift

```diff
@@ -276,3 +291,11 @@ public protocol BuildConfiguration {
   /// #endif
   var compilerVersion: VersionTuple { get }
 }
+
+/// Default implementation of BuildConfiguration, to avoid a revlock with the
+/// swift repo.
```

Comment:
This is not only to avoid revlock with the compiler repo. It is also to not introduce a new requirement to `BuildConfiguration` for an experimental feature that would be API breaking.